### PR TITLE
Scorm hashi bugs

### PR DESCRIPTION
**Pull request by rtibbles**
### Summary
Fixes two small issues that I noticed while using hashi after the SCORM updates:
* When SCORM data is undefined, the SCORM shim will error because it is trying to access an undefined key path
* An option had an extra space inside it, meaning it would not pass validation

### Reviewer guidance
* Check that non-SCORM html5 apps do not give an error trying to access a property on undefined as currently happens

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [x] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: packages/hashi/src/SCORM.js [hunk 1]
```diff
@@ -622,8 +622,8 @@ export default class SCORM extends BaseShim {
       const raw = Number(isNaN(score.raw) ? min : score.raw);
       return Math.max(Math.min((raw - min) / (max - min), 1), 0);
     }
-    const lessonStatus = this.data.cmi.core.lesson_status;
-    if (statusProgressMap.hasOwnProperty(lessonStatus)) {
+    const lessonStatus = getByKeyPath(this.data, 'cmi.core.lesson_status', SCHEMA, self.userData);
+    if (Object.prototype.hasOwnProperty.call(statusProgressMap, lessonStatus)) {
```
Comment (kollivier) [inline packages/hashi/src/SCORM.js:626]: Just curious about why this change was made. e.g. how `Object.prototype.hasOwnProperty.call` differs in behavior from `statusProgressMap.hasOwnProperty`. Is this to handle the case where `lessonStatus` is undefined? If so, any reason not to check it for undefined first?

**Review by kollivier — APPROVED** [review]
Good catch, tested and the problem appears to be fixed!

**Comment by kollivier** [comment]
Also, since pf-mvp changes have been merged to develop, we need to make sure we apply this fix there as well.

File: packages/hashi/src/SCORM.js (hunk 1 above)
Comment (indirectlylit) [reply packages/hashi/src/SCORM.js:626]: This was flagged by our linter which has this rule: https://eslint.org/docs/rules/no-prototype-builtins

I made the same change the last time I merged `pf-mvp` -> `develop`

Comment (rtibbles) [reply packages/hashi/src/SCORM.js:626]: Yeah - I originally made this change on develop, so @indirectlylit's linting fix was caught up in my stash. It seemed simpler to retain this change than to undo it when I reapplied it to pf-mvp.
